lua/languages/global/languages/css.lua

```diff
@@ -3,24 +3,15 @@
 
 local global = require("core.global")
 local languages_setup = require("languages.global.utils")
-local nvim_lsp = require("lspconfig")
 local nvim_lsp_util = require("lspconfig/util")
 local lsp_signature = require("lsp_signature")
 local default_debouce_time = 150
-local lsp_installer = require("nvim-lsp-installer")
-local lsp_installer_servers = require("nvim-lsp-installer.servers")
 
 local language_configs = {}
 
 language_configs["lsp"] = function()
-    local function start_cssls()
-        nvim_lsp.cssls.setup {
-            cmd = {
-                "node",
-                global.lsp_path ..
-                    "lsp_servers/vscode-langservers-extracted/node_modules/.bin/vscode-css-language-server",
-                "--stdio"
-            },
```

Comment:
Providing `cmd` manually is problematic because:

1) It assumes the user's installation directory (this can be configured)
1) These all seem to only target Unix systems, so Windows support is thrown out the window (not sure if Windows support is a goal for this project though)
1) This cmd may very well change over time, which would lead to a drift in what you provide here and what's correct